.github/workflows/build-pr-preview.yml

```diff
@@ -180,6 +180,7 @@ jobs:
       - name: Checkout hashicorp/dev-portal (Frontend)
         uses: actions/checkout@eef61447b9ff4aafe5dcd4e0bbf5d482be7e7871 # v4.2.1
         with:
+          ref: 3f07571e4972d1ade19eea351490039462d20b25
```

Comment:
Only for testing, see dev-portal PR: https://github.com/hashicorp/dev-portal/pull/2721